Classes/common/Encryption/Keychain/CDTEncryptionKeychainStorage.m

```diff
@@ -0,0 +1,198 @@
+//
+//  CDTEncryptionKeychainStorage.m
+//
+//
+//  Created by Enrique de la Torre Fernandez on 12/04/2015.
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import "CDTEncryptionKeychainStorage.h"
+
+#import "CDTLogging.h"
+
+#define CDTENCRYPTION_KEYCHAINSTORAGE_KEY_DOCUMENTID @"CDTDatastoreKey"
```

Comment:
OK but I just realised that this constant is a dictionary value, not a dictionary key; I'm going to change the name to

```
#define CDTENCRYPTION_KEYCHAINSTORAGE_SERVICE_VALUE @"com.cloudant.sync.CDTEncryptionKeychainStorage.service"
```